web/packages/teleport/src/config.ts

```diff
@@ -1056,6 +1059,10 @@ const cfg = {
     );
   },
 
+  getNotificationsUrl(params: UrlNotificationParams) {
+    return generatePath(cfg.api.notificationsPath, { ...params });
```

Comment:
Spreading the params here is actually necessary to avoid a type error related to the index signature, the alternative hacky way would require us to add `[key: string]: string | number | boolean | undefined;` to the params type. I think spreading is cleaner and is what we do everywhere else in this file.